package.json

```diff
@@ -4,62 +4,39 @@
   "description": "RIVER Bullet Journal; CSE110 Spring 2021",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --recursive ./source/javascript/tests/.",
+    
+    "test": "./node_modules/mocha/bin/mocha --recursive --timeout=300000 ./source/javascript/tests",
     "coverage": "nyc npm run test",
-    "lint_tests": "./node_modules/.bin/eslint source/javascript/tests/.",
-    "fix_tests": "./node_modules/.bin/eslint source/javascript/tests/. --fix",
-    "lint_src": "./node_modules/.bin/eslint source/javascript/src/.",
-    "fix_src": "./node_modules/.bin/eslint source/javascript/src/. --fix",
+    "lint_tests": "eslint source/javascript/tests/.",
```

Comment:
Thank you for going back and fixing up any technical debt we had in our build scripts 👍